Port STT.xml

```diff
@@ -188,10 +190,11 @@
 		<book>AET</book>
 		<book>NF</book>
 		<book>KK</book>
+		<book>SW</book>
 	</books>
 	<customdatadirectorynames>
 		<customdatadirectoryname>
-			<directoryname>62aeb98e-e61c-4d0e-b4ff-03e0469e6e2b&gt;1.3</directoryname>
+			<directoryname>Rainforest Carbine Missions Errata</directoryname>

```

Comment:
It took a little GUID digging but that's what that one is. I just... I guess. Fuck it. Whatever.